Components/MapGoogle/functions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Flynt\Components\MapGoogle;
+
+use Flynt\Features\Components\Component;
+use Flynt\Features\Acf\OptionPages;
+
+add_action('wp_enqueue_scripts', function () {
+    $apiKey = OptionPages::get('globalOptions', 'feature', 'Acf', 'googleMapsApiKey');
```

Comment:
Small remark: I know it works either way, but for consistency with using `OptionPages::get` I would like to have all parameters be lowerCamelCase so that, for example, someone looking up the base components doesn't get confused why it's sometimes upper and sometimes lower case.